spec/system/candidate_interface/references/candidate_adding_referees_in_sandbox_spec.rb

```diff
@@ -57,4 +60,21 @@ def then_i_see_that_references_are_given
       expect(all('.govuk-summary-list__value')[4].text).to have_content('Reference given')
     end
   end
+
+  def when_i_provide_incomplete_reference_details
+    visit candidate_interface_references_start_path
+    click_link t('continue')
+    choose 'Academic'
+    click_button t('save_and_continue')
+  end
+
+  def then_i_see_that_the_incomplete_reference_rendered
+    visit candidate_interface_references_review_path
+    within all('.app-summary-card')[2] do
+      expect(all('.govuk-summary-list__value')[0].text).to have_content('Not entered')
+      expect(all('.govuk-summary-list__value')[1].text).to have_content('Not entered')
+      expect(all('.govuk-summary-list__value')[2].text).to have_content('Academic')
+      expect(all('.govuk-summary-list__value')[3].text).to have_content('Not entered')
```

Comment:
Can use the `within_summary_row` [helper](https://github.com/DFE-Digital/apply-for-teacher-training/blob/3261-references-save-at-end-of-flow/spec/support/test_helpers/candidate_helper.rb#L449) to make this slightly clearer.

eg -
`within_summary_row('Reference type') { expect(page.text).to have_content('Academic') }`